StyleCop.Analyzers/StyleCop.Analyzers.Test/SpacingRules/SA1011UnitTests.cs

```diff
@@ -312,6 +311,23 @@ void Method()
             await this.VerifyCSharpDiagnosticAsync(testCode, expected, CancellationToken.None).ConfigureAwait(false);
         }
 
+        [Fact]
+        public async Task TestColonCanFollowSquareBracketWhenPartOfInterpolationFormatClauseAsync()
+        {
+            string testCode = @"
+class ClassName
+{
+    void Method()
+    {
+        var x = new[] { 1, 2, 3, 4, 5, 6, 7 };
+        var t = $""{ x[2]:C}"";
+    }
```

Comment:
Please add a testcase for `$"{x[n],3}"` as well to signify that this is a supported option.

Reply:
➡️ Covered in `TestCommaCanFollowSquareBracketWhenPartOfInterpolationFormatClauseAsync`